<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CodeClassResultsWidget</class>
 <widget class="QWidget" name="CodeClassResultsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Class</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="lstClass"/>
     </item>
     <item>
      <widget class="QLabel" name="lblCode">
       <property name="text">
        <string>Code</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="lstCode">
       <property name="sizeAdjustPolicy">
        <enum>QComboBox::AdjustToContents</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="qf::qmlwidgets::TableView" name="tblView"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qf::qmlwidgets::TableView</class>
   <extends>QTableView</extends>
   <header location="global">qf/qmlwidgets/tableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
